docarray/utils/create_dynamic_doc_class.py

```diff
@@ -49,6 +50,11 @@ class MyDoc(BaseDoc):
     :param model: The input model
     :return: A new subclass of BaseDoc, where every DocList type in the schema is replaced by List.
     """
+    if is_pydantic_v2:
+        raise NotImplementedError(
+            'This method is not supported in Pydantic 2.0. Please use Pydantic 1.8.2 or lower.'
+        )
+
```

Comment:
Why is this?

Reply:
the method we used has been removed from pydantic v2, will have to find a fix later